packages/server/src/prod-server.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Koa from 'koa';
+import koaStatic from 'koa-static';
+import koaConditionalGet from 'koa-conditional-get';
+import koaEtag from 'koa-etag';
+import {fileURLToPath} from 'url';
+import * as pathlib from 'path';
+
+const PORT = process.env['PORT'] || 8080;
+const STATIC_ROOT = pathlib.resolve(
+  fileURLToPath(import.meta.url),
```

Comment:
super nit: might be nice to use `fileURLToPath(new URL('../../../context/_site', import.meta.url))`

Reply:
Done (removes need for path library import :-1: )